EasyLayout.Forms/EasyLayout.cs

```diff
@@ -341,103 +337,110 @@ public static void ConstrainLayout(this RelativeLayout relativeLayout, Expressio
         {
-            return GetSingleAssertionOrDefault(assertions, i => i.IsExplicitWidthAssertion(), "Multiple width assertions found");
+            return GetSingleAssertionOrDefault(assertions, i => i.IsExplicitWidthAssertion(),
+                "Multiple width assertions found");
         }
 
         private static Assertion GetRelativeHeightConstraint(IGrouping<View, Assertion> assertions)
         {
-            var top = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Top, "Only one top assertion allowed");
-            var bottom = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Bottom, "Only one bottom assertion allowed");
-			if (top == null || bottom == null) return null;
-			return bottom;
-		}
+            var top = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Top,
+                "Only one top assertion allowed");
+            var bottom = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Bottom,
+                "Only one bottom assertion allowed");
+            if (top == null || bottom == null) return null;
+            return bottom;
+        }
 
-        private static Assertion GetRelativeWidthConstraint(IGrouping<View, Assertion> assertions) 
+        private static Assertion GetRelativeWidthConstraint(IGrouping<View, Assertion> assertions)
         {
-            var left = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Left, "Only one left assertion allowed");
-			var right = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Right, "Only one left assertion allowed");
+            var left = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Left,
+                "Only one left assertion allowed");
+            var right = GetSingleAssertionOrDefault(assertions, i => i.Position == Position.Right,
+                "Only one left assertion allowed");
             if (left == null || right == null) return null;
             right.UpdateMarginFromRelativeWidthConstraint(left);
             return right;
-		}
+        }
 
-        private static Assertion GetSingleAssertionOrDefault(IGrouping<View, Assertion> viewsGroupedByRule, Func<Assertion, bool> func, string errorMessage)
+        private static Assertion GetSingleAssertionOrDefault(IGrouping<View, Assertion> viewsGroupedByRule,
+            Func<Assertion, bool> func, string errorMessage)
         {
             var assertions = viewsGroupedByRule.Where(func).ToList();
-            if (assertions.Count > 1)
-            {
-                throw new ArgumentException(errorMessage + ".  Element name: " + viewsGroupedByRule.First().LeftName);
-            }
-            if (assertions.Count == 0) return null;
-            return assertions[0];
+            return assertions.Count > 1
+                ? throw new ArgumentException(errorMessage + ".  Element name: " + viewsGroupedByRule.First().LeftName)
```

Comment:
Dude, I can live with a lot of the edits here, but this is just going too far.  How is doubly nested ternaries possibly an improvement to anything ever?  How is throwing an exception within the body of a ternary possibly improving things?  Please revert this.